qa/suites/deepsea/tier2/add-node/2-test-phase.yaml

```diff
@@ -0,0 +1,13 @@
+tasks:
+        - exec:
+                client.salt_master:
+                        - 'ceph -s'
```

Comment:
before adding a node, we should run some kind of validation routine that checks the number of nodes and compares that to "expected number of nodes", which could be given as a configuration parameter here in the yaml